src/domain/account/errors/account-save.error.ts

```diff
@@ -1,4 +1,4 @@
-export class EmailSaveError extends Error {
+export class AccountSaveError extends Error {
   constructor(chainId: string, safeAddress: string, account: string) {
     super(
       `Error while saving provided email was not saved. chainId=${chainId}, safeAddress=${safeAddress}, account=${account}`,
```

Comment:
Here too if we change to `createAccount`.

Reply:
[fa63a4e](https://github.com/safe-global/safe-client-gateway/pull/1050/commits/fa63a4e9950036c1b044c936774359b805c7c62f)

Reply:
The message needs to be tweaked a bit:

```suggestion
      `Error while creating account. Account was not created. chainId=${chainId}, safeAddress=${safeAddress}, signer=${signer}`,
```